config/config.ts

```diff
@@ -2,7 +2,13 @@ import { defineConfig } from 'umi';
 import defaultSettings from './defaultSettings';
 import proxy from './proxy';
 
-const { REACT_APP_ENV } = process.env;
+const { REACT_APP_ENV, NETLIFY_DEMO, NODE_ENV } = process.env;
+
+const getRequestPrefix = () => {
+  if (NETLIFY_DEMO) return '/api';
```

Comment:
it seems not to work on Netlify...

![image](https://user-images.githubusercontent.com/2106987/82560893-e1980680-9ba4-11ea-92c0-aef296f0886e.png)

Reply:
![image](https://user-images.githubusercontent.com/2106987/82560915-eeb4f580-9ba4-11ea-8fc5-26a8e619fffb.png)

Reply:
https://deploy-preview-208--apisix-dashboard.netlify.app/dashboard/settings

Reply:
In fact, I think we can directly request the Admin API at the browser without using netlifly's 
request forwarding.